catalog-rest-service/src/main/resources/json/schema/metadataIngestion/databaseServiceMetadataPipeline.json

```diff
@@ -3,6 +3,114 @@
   "$schema": "http://json-schema.org/draft-07/schema#",
   "title": "DatabaseServiceMetadataPipeline",
   "description": "DatabaseService Metadata Pipeline Configuration.",
+  "definitions": {
+    "credentials": {
+      "description": "GCS Credentials.",
```

Comment:
We could move the credentials to json/schema/security/credentials. this way we can model out AWS etc at some other point

Reply:
okay